downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc

```diff
@@ -41,7 +41,7 @@ If this field is not set to "/var/log/awx", then follow these configuration step
 + 
 Note that this change will need to be made on each {ControllerName} in a load-balanced scenario.
 
-All user session data must be logged in order to support the troubleshooting, debugging and forensic analysis for visibility and analytics. Without this data from the controller’s web server, important auditing and analysis for event investigations will be lost. To modify or verify this is configured in the system, follow these steps:
+All user session data must be logged to support troubleshooting, debugging and forensic analysis for visibility and analytics. Without this data from the controller’s web server, important auditing and analysis for event investigations will be lost. To modify or verify that the system is configured to ensure that user session data is logged, use the following steps:
```

Comment:
"To modify or verify that the system is configure to ensure that user session data is logged" reads a little weird to me. Maybe just rephrase it to "To verify that the system...". Essentially just remove the "modify or" from that sentence.

Reply:
Done. Thank you!